lib/java-wrapper.ts

```diff
@@ -22,7 +22,7 @@ function getCallGraphGenCommandArgs(
     '-cp',
     jarPath,
     'io.snyk.callgraph.app.App',
-    '--application-classpath',
+    '--application-classpath-file',
     classPath,
```

Comment:
Maybe change this as well to be `classPathFile` ?

Reply:
This depends on the callgraph-generator param and it was done this way.